go-controller/pkg/ovn/ovn.go

```diff
@@ -318,6 +322,10 @@ func (oc *Controller) Run(wg *sync.WaitGroup) error {
 
 	klog.Infof("Completing all the Watchers took %v", time.Since(start))
 
+	// Remove old multicastDefaultDeny port group now that all ports
```

Comment:
why dont we just use one port group with mp0 ports in it for the reject acls, and leave multicast alone?

Reply:
If we deny the multicast traffic on management port, both the port-group will be same right? so do we need two port-group in that case?

Reply:
the problem becomes on upgrade the port group now has a different name, so we have to delete the old one, and then wait for all the pod events to add their ports to the new one. But I guess per our discussion there's no other good option if we want to rename the current port group.

Reply:
thanks for summarizing it tim.